### ISSUE TITLE
Adding release tag

### ISSUE THREAD

**Issue by hsmaan** [mojaveazure/seurat-disk#57]
Hi,

I was wondering if it's possible to add a release tag for the repo to build with conda. The package is not on a conda channel yet and building using `conda skeleton cran https://github.com/mojaveazure/seurat-disk` fails because of the lack of a tag and results in -  `Error: no tags found`. The problem is detailed by [this issue on the conda repo](https://github.com/conda/conda/issues/6674). Thank you.

**Comment by tessadgreen** [comment]
I was able to install seurat-disk directly from RStudio in a session launched from conda. This is surely the path to a future broken environment, but it works for now. 

As a temporary fix, I also forked the current version and added a release tag (https://github.com/tessadgreen/seurat-disk), which does make it compatible with conda skeleton, though I didn't end up needing that method. I'm reluctant to make a pull request for it, given that @mojaveazure probably wants to manage their own releases, but it might work as a stopgap.
